src/jwt/decode.js

```diff
@@ -7,7 +7,7 @@ export default async (driver, authorizationHeader) => {
   try {
     const decoded = await jwt.verify(token, process.env.JWT_SECRET)
     id = decoded.sub
-  } catch {
+  } catch (e) {
```

Comment:
e could also mean event, as a good practice use err or error instead.

Reply:
Yeah, I will change that, and how was that merged to master with a syntax error merged.

Reply:
That’s a really good question.